src/hotspot/cpu/ppc/stubGenerator_ppc.cpp

```diff
@@ -631,6 +631,135 @@ class StubGenerator: public StubCodeGenerator {
     return start;
   }
 
+// Generate stub for ghash process  blocks.
```

Comment:
There are multiple double-whitespaces in the new comments. Please clean them up!